### ISSUE TITLE
hyphenation dictionary

### ISSUE THREAD

**Issue by wimmuskee** [OpenTaal/opentaal-hunspell#9]
The previous version (2.10-g) also contained a hyphenation dictionary. Are there plans to include that again in 2.20+ ?

**Comment by PanderMusubi** [comment]
Yes, towards the end of this year. The current hyphenation patters are from before the last spelling change, so there is a lot of work to upgrade these.

**Comment by AEApple** [comment]
Do you still expect the updated hyphenation patterns to be available soon?

**Comment by de-wolff** [comment]
I saw that there was a beta version available. However, there are a lot of words that were not, and still are not hyphenated
Attached a file with first the hyphenation according to hyph_nl.dic, and second what it should be in my opinion.
For the record: It is much better now. 
[dictionary.txt](https://github.com/OpenTaal/opentaal-hunspell/files/7777438/dictionary.txt)

**Comment by PanderMusubi** [comment]
Where did you see the beta version? We did not publish the new one yet. Will take your additions as input for the new version too. This will be released in January.
